### PR TITLE
Fix inconsistent spacing in the doc

### PR DESCRIPTION
**Pull request by reyang**
Was looking at the readme file and noticed the nits, trying to fix them.

### CODE REVIEW

**Review by ThomsonTan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -27,15 +27,14 @@ of the current project.
...
 
-* ubuntu-18.04 ( Default GCC Compiler - 7.5.0 )
-* ubuntu-18.04 ( GCC 4.8 with -std=c++11 flag )
-* ubuntu-20.04 ( Default GCC Compiler - 9.3.0 with -std=c++20 flags )
-* macOS 10.15 ( Xcode 12.2 )
-* Windows Server 2019 (Visual Studio Enterprise 2019 )
+* ubuntu-18.04 (Default GCC Compiler - 7.5.0)
+* ubuntu-18.04 (GCC 4.8 with -std=c++11 flag)
+* ubuntu-20.04 (Default GCC Compiler - 9.3.0 with -std=c++20 flags)
+* macOS 10.15 (Xcode 12.2 )
```
Comment (lalitb) [inline README.md:33]: nit: we can remove spacing here too after 12.2 ?

**Review by lalitb — APPROVED** [review]
Thanks for the fix.

**Review by jsuereth — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (reyang) [reply README.md:33]: Sorry I missed it. I will send another PR enforcing markdownlint in the CI and also fixing this nit.

Comment (reyang) [reply README.md:33]: Sent the follow up PR #578.

**Pull request merged**